### ISSUE TITLE
pass collection object to the Model::insert()

### ISSUE THREAD

**Issue by mobayen** [laravel/ideas#2099]
it would be nice if the insert() method accepts a collection object as well!

right now it works like this:
`Model::insert($CollectionObj->toArray());`

my suggestion :
`Model::insert($CollectionObj);`

**Comment by ryangjchandler** [comment]
I don't think this is worth it. Behind the scenes, it would just call the toArray() method anyway, just do it yourself.

**Comment by mobayen** [comment]
maybe, maybe not!
the thing is we are dealing with collections all around the project, and many methods work great with them but there are a couple of them that don't!
like `Model::insert()` or `Validator::make()`
it is nice if they accept collections as well. 
besides, that makes Laravel more predictable and united

anyways, I am just suggesting and it's up to @taylorotwell
